playbooks/files/loki/config.yml.j2

```diff
@@ -0,0 +1,55 @@
+auth_enabled: false
+
+server:
+  http_listen_port: 3100
+  grpc_listen_port: 9096
+  log_level: info
+  grpc_server_max_concurrent_streams: 1000
+
+common:
+  instance_addr: 0.0.0.0
+  path_prefix: /tmp/loki
+  storage:
+    filesystem:
+      chunks_directory: /tmp/loki/chunks
+      rules_directory: /tmp/loki/rules
+  replication_factor: 1
+  ring:
+    kvstore:
+      store: inmemory
+
+limits_config:
+  ingestion_rate_mb: {{ loki__limits_config__ingestion_rate_mb }}
+  ingestion_burst_size_mb: {{ loki__limits_config__ingestion_burst_size_mb }}
```

Comment:
The loki configuration is incredibly complex, badly designed and documented.. It might make sense to hardcode these too until we understand loki better and then we can try to allow a more comprehensive customisation